pandas/core/arrays/base.py

```diff
@@ -708,6 +712,25 @@ def _add_comparison_ops(cls):
         cls.__le__ = cls._create_comparison_method(operator.le)
         cls.__ge__ = cls._create_comparison_method(operator.ge)
 
+    def _reduce(self, name, skipna=True, **kwargs):
```

Comment:
What sort of things go into kwargs? Is it things like `ddof`?

Reply:
yes, they are specific to the reducer